fe/fe-core/src/main/java/com/starrocks/catalog/Column.java

```diff
@@ -718,7 +718,7 @@ public boolean equals(Object obj) {
         if (!this.getType().equals(other.getType())) {
             return false;
         }
-        if (this.aggregationType != other.getAggregationType()) {
+        if (!AggregateType.equals(this.aggregationType, other.getAggregationType())) {
```

Comment:
I don't think it is a good idea that equals return true when comparing null and NONE.
You can provide a normalize function to convert null to None, I think this will be better
And we can still make difference between null and NONE